cpp/ql/test/query-tests/Security/CWE/CWE-119/semmle/tests/test_buffer_overrun.cpp

```diff
@@ -0,0 +1,39 @@
+#include <stdint.h>
```

Comment:
We normally don't include standard headers in our tests. Instead, you should write your own versions of exactly what you need. For this specific case, I guess it will be enough to add your own typedef for `uint8_t`. So you should replace this line with something like:
```c
typedef unsigned char uint8_t;
```

Reply:
No worries - I've replaced the include with a typedef.

Reply:
The relevant documentation is here: https://github.com/github/codeql/blob/main/cpp/ql/test/README.md